app/src/main/java/org/feedhenry/welcome/fragments/LocationFragment.java

```diff
@@ -243,26 +234,21 @@
 	@Override
 	public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 							 @Nullable Bundle savedInstanceState) {
-		View view = inflater.inflate(R.layout.fragment_location, null);
+		View view = inflater.inflate(R.layout.fragment_location, container, false);
 
 		toggle = view.findViewById(R.id.weather_toggle);
 
-		weatherIcon = (ImageView) view.findViewById(R.id.weather_icon);
-		weatherDesc = (TextView) view.findViewById(R.id.weather_desc);
-		weatherDate = (TextView) view.findViewById(R.id.weather_date);
-		weatherLow = (TextView) view.findViewById(R.id.weather_low);
-		weatherHigh = (TextView) view.findViewById(R.id.weather_high);
+		weatherIcon = view.findViewById(R.id.weather_icon);
```

Comment:
Know this is probably the linter, but just wondering if there's any potential repercussions of removing the cast

Reply:
I think that it's not possible to use the template with older gradle plugin/buildtools.

from Android docs:
> *Note:* In most cases -- depending on compiler support -- the resulting view is automatically cast to the target class type. If the target class type is unconstrained, an explicit cast may be necessary.